### PR TITLE
made populate initial tag to filter out stale ones outside of given t…

### PR DESCRIPTION
**Pull request by heejaechang**
…ext range.

fix https://github.com/dotnet/roslyn/issues/9613

### CODE REVIEW

**Comment by heejaechang** [comment]
@dotnet/roslyn-analysis @dotnet/roslyn-ide can you take a look?

**Comment by heejaechang** [comment]
test vsi please

File: src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsTaggerProvider.AggregatingTagger.cs [hunk 1]
```diff
@@ -65,10 +67,45 @@ private void GetInitialDiagnostics(CancellationToken cancellationToken)
...
+                        }
+
+                        OnDiagnosticsUpdated(
+                            DiagnosticsUpdatedArgs.DiagnosticsCreated(
+                                updateArgs.Id, updateArgs.Workspace, project.Solution, updateArgs.ProjectId, updateArgs.DocumentId, diagnostics));
+                    }
+                }
+            }
+
+            private ImmutableArray<DiagnosticData> GetInitialDiagnostics(Solution solution, UpdatedEventArgs args, CancellationToken cancellationToken)
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsTaggerProvider.AggregatingTagger.cs]: I would call this "AdjustInitialDiagnostics".  And explain in the code that hte initial diagnostics may not match the source text.

**Comment by CyrusNajmabadi** [comment]
:+1:

**Comment by heejaechang** [comment]
test vsi please

**Pull request merged**